NorthstarDLL/util/version.cpp

```diff
@@ -46,7 +46,8 @@ void InitialiseVersion()
 	if (ntdll)
 	{
 		// real win32 version info (i.e., ignore manifest)
-		DWORD(WINAPI * RtlGetVersion)(LPOSVERSIONINFOEXW);
+		DWORD(WINAPI * RtlGetVersion)
+		(LPOSVERSIONINFOEXW);
```

Comment:
Umm

Reply:
Seems to be caused by `ColumnLimit: 0`, removing `ColumnLimit` completely seems to default it to something, but fixes this